coala_quickstart/Strings.py

```diff
@@ -50,3 +50,9 @@
 
 You can see all of them here: {}
 """.format(BEAR_DOCS_URL)
+
+GENERATOR_INFO = """
+# coala-quickstart is a tool that helps users to quickly get started with coala.
+# It can generate a .coafile (coala's configuration file) that is tailored
+# perfectly for your project without any interaction.
```

Comment:
Yeah i guess @javydb is right, GENERATOR_INFO is not needed. Plus anyone using it will already know all this. Anyways i had just copied it from the coala-quickstart github page :stuck_out_tongue: